winit/src/settings.rs

```diff
@@ -90,6 +90,12 @@ impl Window {
                 .with_max_inner_size(winit::dpi::LogicalSize { width, height });
         }
 
+        #[cfg(target_os = "linux")]
```

Comment:
Please match the cfg winit uses for `winit::platform::unix`:

```rust

#![cfg(any(
    target_os = "linux",
    target_os = "dragonfly",
    target_os = "freebsd",
    target_os = "netbsd",
    target_os = "openbsd"
))]
```

Reply:
just did it and rebased master